utils/apps/cassandra/nodetool_status.yml

```diff
@@ -0,0 +1,22 @@
+## CHECKING FOR THE DISTRIBUTION OF THE LOAD ON THE RING
+- name: checking for the distribution of the load on the ring
+  shell: >
+    kubectl exec -it --namespace {{ app_ns }} $(kubectl get pods --namespace {{ app_ns }} -l {{ app_label }} -o jsonpath='{.items[0].metadata.name}') nodetool status | awk '{print $6}'
```

Comment:
Put this command in multiple lines

Reply:
Can we see if there is a dedicated cassandra client we can use here from an external pod -- it can also serve as an external liveness check.